SDWebImage/Core/SDAnimatedImage.m

```diff
@@ -335,7 +335,7 @@ - (NSUInteger)sd_memoryCost {
 @implementation SDAnimatedImage (Metadata)
 
 - (BOOL)sd_isAnimated {
-    return YES;
+    return self.animatedImageFrameCount > 1;
```

Comment:
That https://github.com/SDWebImage/SDWebImageYYPlugin need to update with this behavior as we..